osutil/disks/disks_linux.go

```diff
@@ -447,52 +434,36 @@ func diskFromPartUdevPropsAndOpts(props map[string]string, opts *Options) (*disk
 		if err != nil {
 			return nil, fmt.Errorf(errFmt, err)
 		}
+		dmUUID = bytes.TrimSpace(dmUUID)
 
 		dmName, err := ioutil.ReadFile(filepath.Join(dmDir, "name"))
 		if err != nil {
 			return nil, fmt.Errorf(errFmt, err)
 		}
+		dmName = bytes.TrimSpace(dmName)
+
+		matchedHandler := false
+		for _, h := range deviceMapperHandlers {
+			if dev, devTypeOpt, ok := h(dmUUID, dmName); ok {
+				var err error
+				if devTypeOpt == "name" {
+					props, err = udevPropertiesForName(dev)
+				} else if devTypeOpt == "path" {
```

Comment:
do we have a use case for "path"? it seems all we have now uses "name" ?

Reply:
No not right now, we could leave it out since we only use name currently

Reply:
this is removed in https://github.com/snapcore/snapd/pull/11035